src/renderers/WebGLRenderer.js

```diff
@@ -2469,7 +2474,7 @@ function WebGLRenderer( parameters ) {
 
 		if ( _currentFramebuffer !== framebuffer ) {
 
-			_gl.bindFramebuffer( _gl.FRAMEBUFFER, framebuffer );
+			_gl.bindFramebuffer( optType || _gl.FRAMEBUFFER, framebuffer );
```

Comment:
Thinking to move this `.setRenderTarget()` update to another PR because this's for WebGL2.0 specific.

Reply:
Yep I agree with that approach

Reply:
Reverted. This change should be discussed well in another PR.